operators/src/processing/raster_vector_join/util.rs

```diff
@@ -137,6 +143,10 @@ impl CoveredPixels<MultiPoint> for MultiPointCoveredPixels {
     fn collection(self) -> FeatureCollection<MultiPoint> {
         self.collection
     }
+
+    fn covered_spatial_bounds(&self) -> Option<BoundingBox2D> {
+        self.collection.bbox()
```

Comment:
I am not sure if this is correct. This is the bbox of the whole collection, but we are always looking at single tiles, which may not cover the whole collection.